### PR TITLE
[Fix] `react/jsx-no-bind` Add props to example

### PR DESCRIPTION
**Pull request by ronanmathew**
https://reactjs.org/blog/2015/01/27/react-v0.13.0-beta-1.html#autobinding

### CODE REVIEW

File: docs/rules/jsx-no-bind.md [hunk 1]
```diff
@@ -121,8 +121,8 @@ Unfortunately [React ES6 classes](https://facebook.github.io/react/blog/2015/01/
 
 ```jsx
 class Foo extends React.Component {
-  constructor() {
-    super();
+  constructor(props) {
```
Comment (ljharb) [inline docs/rules/jsx-no-bind.md]: This isn't actually needed, and to be fully correct, it'd be `props, context`. It's a fine change tho.

Comment (ronanmathew) [reply docs/rules/jsx-no-bind.md]: While not strictly necessary it allows use of `this.props` in the constructor and is suggested best practice by the React docs:

"Class components should always call the base constructor with props."
- https://reactjs.org/docs/state-and-lifecycle.html#adding-local-state-to-a-class

Comment (ljharb) [reply docs/rules/jsx-no-bind.md]: In that case, what about `constructor(...args) { super(...args);` ? that way people copying the example will always forward all args.

Comment (ronanmathew) [reply docs/rules/jsx-no-bind.md]: Comments addressed @ljharb. Anything else or are you happy to merge?

**Pull request merged**